b2share/modules/schemas/serializers.py

```diff
@@ -110,3 +134,86 @@ def community_schema_to_json_serializer(community_schema, code=200,
     if headers is not None:
         response.headers.extend(headers)
     return response
+
+
+def new_schema_to_json_serializer(schema, code=200, headers=None):
```

Comment:
Or rather, I don't see any difference with `block_schema_to_json_serializer`. Why not reuse it?